napari/layers/_tests/test_dask_layers.py

```diff
@@ -34,6 +35,7 @@ def mock_set_view_slice():
     dask.__version__ = _dask_ver
 
     # make sure we can resize the cache
+    utils.resize_dask_cache(10000)
     assert utils.dask_cache.cache.total_bytes > 1000
     utils.resize_dask_cache(1000)
     assert utils.dask_cache.cache.total_bytes <= 1000
```

Comment:
```suggestion
    utils.resize_dask_cache(10000)
    assert utils.dask_cache.cache.available_bytes == 10000
```

ah, the issue here is just the difference between `available_bytes` and `total_bytes`... with the former giving you the max cache size, and the latter giving you the current number of bytes.  The previous test was asserting that if you resize the cache, it will actively discard bytes and recover memory... now we just want to make sure that resizing expands the amount available, right?

Reply:
Ah of course. 🤦‍♂️ thanks @tlambert03 . I'll push a change in the morning.